lib/elixir/lib/kernel.ex

```diff
@@ -8,33 +8,29 @@ import :elixir_bootstrap
 
 defmodule Kernel do
   @moduledoc """
-  This module is the entry point of the Elixir programming language.
-
-  It provides the default functions and macros Elixir imports into your
-  environment. Those can be directly invoked in your code without the
-  module name:
-
+  `Kernel` is Elixir's set of core functionality.
+  
+  It mainly consists of:
+  
+    * syntactic sugar - for more readable and intuitive code
+    * [macros](https://elixir-lang.org/getting-started/meta/macros.html) - for defining new functionality
+    * [guard](guards.html) checks - for more powerful function definitions
+    
+  `Kernel` is automatically imported into all `.ex` and `.exs` files,
```

Comment:
Should we say that `Kernel` is automatically imported in all Elixir code, instead of just `.ex` and `.exs` files?